src/pykeen/models/uncertainty.py

```diff
@@ -203,7 +203,7 @@ def predict_hrt_uncertain(
     Example Usage::
 
         from pykeen.pipeline import pipeline
-        from pykeen.models.predict import predict_hrt_uncertain
+        from pykeen.models.uncertainty import predict_hrt_uncertain
```

Comment:
does this module now rely on `pykeen.predict`? sort of complicated, maybe this module should also be in a predict subpackage?

Reply:
I am unsure about this :sweat_smile: 

The uncertainty module only wraps the model's `predict_{h,r,t}`, but does not offer the flexibility of the general predict module.

Reply:
okay then leave it for now